script/numenta_benchmark.py

```diff
@@ -0,0 +1,135 @@
+import psycopg2
+import sys
+import os
+import os.path
+import csv
+import dateutil.parser
+from datetime import datetime
+import itertools
+import re
+import json
+
+# Note: initially need to run `python scripts/create_new_detector.py --detector macrobase_ma`
```

Comment:
That's a good question. slf4j would have been ideal since it's already being used, but unfortunately it's buggy. The options I see are: (i) keep the nab branch, updating as necessary. I don't think it's a huge deal if it gets out of date. (ii) Adding another flag. I'm not a fan of having a ton of different flags, but if we want to use this longer term it would probably be the better approach.